src/sections/app.style.js

```diff
@@ -208,6 +208,12 @@ section{
     }
 }
 
+.docker-extension-meshery {
```

Comment:
looks good @SooditK, but why did you added the styles to the global?, Ideally you should add it to the styles file of docker-extension-meshery-styles https://github.com/layer5io/layer5/blob/master/src/sections/Docker-Meshery/dockerMeshery.style.js